android/settings.gradle

```diff
@@ -6,95 +6,42 @@ project(':ReactAndroid').projectDir = new File(rootDir, 'ReactAndroid')
-include ":expo-face-detector"
-project(":expo-face-detector").projectDir = new File(rootDir, "../modules/expo-face-detector/android")
-
-include ":expo-file-system-interface"
-project(":expo-file-system-interface").projectDir = new File(rootDir, "../modules/expo-file-system-interface/android")
-
-include ":expo-file-system"
-project(":expo-file-system").projectDir = new File(rootDir, "../modules/expo-file-system/android")
-
-include ":expo-payments-stripe"
-project(":expo-payments-stripe").projectDir = new File(rootDir, "../modules/expo-payments-stripe/android")
-
-include ":expo-permissions-interface"
-project(":expo-permissions-interface").projectDir = new File(rootDir, "../modules/expo-permissions-interface/android")
-
-include ":expo-permissions"
-project(":expo-permissions").projectDir = new File(rootDir, "../modules/expo-permissions/android")
-
-include ":expo-sensors-interface"
-project(":expo-sensors-interface").projectDir = new File(rootDir, "../modules/expo-sensors-interface/android")
-
-include ":expo-sensors"
-project(":expo-sensors").projectDir = new File(rootDir, "../modules/expo-sensors/android")
-
-include ":expo-react-native-adapter"
-project(":expo-react-native-adapter").projectDir = new File(rootDir, "../modules/expo-react-native-adapter/android")
-
-include ":expo-sms"
-project(":expo-sms").projectDir = new File(rootDir, "../modules/expo-sms/android")
-
-include ":expo-print"
-project(":expo-print").projectDir = new File(rootDir, "../modules/expo-print/android")
-
-include ":expo-media-library"
-project(":expo-media-library").projectDir = new File(rootDir, "../modules/expo-media-library/android")
-
-include ":expo-analytics-segment"
-project(":expo-analytics-segment").projectDir = new File(rootDir, "../modules/expo-analytics-segment/android")
-
-include ":expo-font"
-project(":expo-font").projectDir = new File(rootDir, "../modules/expo-font/android")
-
-include ":expo-font-interface"
-project(":expo-font-interface").projectDir = new File(rootDir, "../modules/expo-font-interface/android")
-
-include ":expo-location"
-project(":expo-location").projectDir = new File(rootDir, "../modules/expo-location/android")
-
-include ":expo-barcode-scanner"
-project(":expo-barcode-scanner").projectDir = new File(rootDir, "../modules/expo-barcode-scanner/android")
-
-include ":expo-barcode-scanner-interface"
-project(":expo-barcode-scanner-interface").projectDir = new File(rootDir, "../modules/expo-barcode-scanner-interface/android")
-
-include ":expo-contacts"
-project(":expo-contacts").projectDir = new File(rootDir, "../modules/expo-contacts/android")
-
-include ":expo-ads-admob"
-project(":expo-ads-admob").projectDir = new File(rootDir, "../modules/expo-ads-admob/android")
-
-include ":expo-local-authentication"
-project(":expo-local-authentication").projectDir = new File(rootDir, "../modules/expo-local-authentication/android")
+def modulesDir = new File(rootDir, "../packages")
+
+def includeUniversalModule = { name ->
+  include ":$name"
```

Comment:
are there any static analysis tools that would break with this dynamic include?

Reply:
None that I can think of.